Tests/test_file_icns.py

```diff
@@ -36,6 +35,22 @@ def test_save(self):
         self.assertEqual(reread.size, (1024, 1024))
         self.assertEqual(reread.format, "ICNS")
 
+    @unittest.skipIf(sys.platform != 'darwin', "requires MacOS")
+    def test_save_append_images(self):
+        im = Image.open(TEST_FILE)
+
+        temp_file = self.tempfile("temp.icns")
+        providedIm = Image.new('RGBA', (32, 32), (255, 0, 0, 0))
```

Comment:
Shall we follow conventions and name it something like `provided_im`?

Reply:
Okay, sure. I've made that change.